### ISSUE TITLE
`task run --follow` doesn’t show all task output

### ISSUE THREAD

**Issue by aviflax** [aws/copilot-cli#1391]
The first N (~100?) lines of output are usually not output by the tool, so I generally use the ECS console to view the task logs. It’d be better if the tool would (if possible) print all the output.

**Comment by iamhopaul123** [comment]
Hello @aviflax, could you elaborate the issue: what does it show on the ECS console vs. by Copilot (with the timestamp would be great)?

edited: if it swallows the first N lines of log then it sounds like a bug to me.

**Comment by aviflax** [comment]
Sure, here’s the bottom of the output from copilot:

```
✔ Task integration-tests is running.

copilot-task/integration-                     "stack": "TypeError: dateObj.getTime is not a function <SNIP>
copilot-task/integration-                     "statusCode": 500
copilot-task/integration-                   }
copilot-task/integration-                 }
copilot-task/integration-             
copilot-task/integration-              code:
copilot-task/integration-                 want 200, received 500
copilot-task/integration-             
copilot-task/integration- successful transactions: 1/2
copilot-task/integration- failed transactions:    1/2
Task has stopped.
```

And here’s the full output in the console — I copied it from the CloudWatch console, which has a _copy to clipboard_ feature, but the same lines show up in the ECS console.

```csv
timestamp,message
1600289678011,data-15min:
1600289678011,    OK   get
1600289678027,weather-history-not-in-db:
1600289678027,    FAIL get
1600289678027,"         request: GET http://pienado.plotwatt.local/api/history?filter={""where"":{""and"":[{""requestedDate"":{""gt"":""2020-01-01T00:00:00Z""}},{""requestedDate"":{""lt"":""2020-01-01T01:00:00Z""}},{""latitude"":""34.0903""},{""longitude"":""-118.3447""}]}} "
1600289678027,         errors:
1600289678027,             body:
1600289678027,                expected value for body doesn't match actual:
1600289678027,                want:
1600289678027,"                ""foo
"""
1600289678027,                received:
1600289678027,                {
1600289678027,"                  ""error"": {"
1600289678027,"                    ""message"": ""dateObj.getTime is not a function"","
1600289678027,"                    ""name"": ""TypeError"","
1600289678027,"                    ""stack"": ""TypeError: dateObj.getTime is not a function <SNIP>"
1600289678027,"                    ""statusCode"": 500"
1600289678027,                  }
1600289678027,                }
1600289678027,            
1600289678027,             code:
1600289678027,"                want 200, received 500"
1600289678027,            
1600289678027,successful transactions: 1/2
1600289678027,failed transactions:     1/2
```

**Comment by iamhopaul123** [comment]
Got it. This bug should've been fixed by #1328 which happened after the last release. Next week we are going to make another release. Please check in again next week after the release to see if the bug goes away!

**Comment by aviflax** [comment]
Good to hear — thanks!

Are there any pre-release builds I could try, if they’re generally pretty stable?

**Comment by iamhopaul123** [comment]
Yeah you could simply run `make build` if you pull our repo and cd into it. Also this guide might help https://github.com/aws/copilot-cli/blob/mainline/CONTRIBUTING.md

**Comment by efekarakus** [comment]
Heya @aviflax the fix was released in [v0.4.0](https://github.com/aws/copilot-cli/releases/tag/v0.4.0)! Let us know if you run into the issue again

**Comment by aviflax** [comment]
Working great -- thanks!
